### ISSUE TITLE
Creating an emotion check-in feature

### ISSUE THREAD

**Issue by RobotsBuildingEducation** [RobotsBuildingEducation/RobotsBuildingEducation#24]
![image](https://github.com/RobotsBuildingEducation/RobotsBuildingEducation/assets/65219666/3fe3aeef-d158-4c02-865c-f358019a973a)

Product inspiration: https://howwefeel.org/
<img width="1507" alt="image" src="https://github.com/RobotsBuildingEducation/RobotsBuildingEducation/assets/65219666/72166424-e2ae-44d2-b359-bbdfb46b2625">

So this feature can be implemented in a number of ways and being web-only right now definitely adds a layer of challenge.

One feature that Duolingo uses often to increase engagement is by notifying user's often to keep trying. So this feature is inspired by both Duolingo and How We Feel.

Simply put, I want to find a way to regularly notify users to check in with their emotions and to simply start adding more mental health and care features. I think it would be cool to allow users to input how they feel and see private data about user's emotional state in general.

I think this feature will result in health encouragement and potentially help users connect and feel less alone in their challenges and experiences.

<hr />

Given that this feature is rather long, if you decide to work on it, please take ownership of it since it is revenue generating software that you can use in other places too.
